### PR TITLE
EIP-3361: New JSON-RPC method for signing Ethereum messages (eth_signMessage)

### PR DESCRIPTION
**Pull request by pedrouid**
This proposal aims to reduce the fragmentation of both decentralizated applications and wallets regarding JSON-RPC methods to request message signatures. Also aims to incentivize implementers to move away from legacy methods that may have inherited broken implementations.

### CODE REVIEW

File: EIPS/eip-xxxx.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+eip: <to be assigned>
```
Comment (lightclient) [inline EIPS/eip-xxxx.md:2]:
```suggestion
eip: 3361
```

File: EIPS/eip-xxxx.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+title: JSON-RPC method to sign Ethereum messages (eth_signMessage)
+author: Pedro Gomes <@pedrouid>
```
Comment (lightclient) [inline EIPS/eip-xxxx.md:4]:
```suggestion
author: Pedro Gomes (@pedrouid)
```

File: EIPS/eip-xxxx.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+discussions-to: <to be assigned>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-03-10
+requires: 191, 1474
+---
+
+## Simple Summary
+Introduces a new JSON-RPC method name to request Ethereum message signatures in light of eth_sign and personal_sign history
```
Comment (lightclient) [inline EIPS/eip-xxxx.md:14]: I'm not sure what "in light" means here?

File: EIPS/eip-xxxx.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    "jsonrpc": "2.0",
+    "result": "0xa3f20717a250c2b0b729b7e5becbff67fdaef7e0699da4de7ca5895b02a170a12d887fd3b17bfdce3481f10bea41f45ba9f709d39ce8325427b57afcfc994cee1b"
+}
+```
+
+## Rationale
+The specification maintains the parameters of the standardized eth_sign method with the prefixed signature behavior using a new method name that is less ambigious.
+
+## Backwards Compatibility
+
```
Comment (lightclient) [inline EIPS/eip-xxxx.md:66]:
```suggestion
```

**Review by lightclient — CHANGES_REQUESTED** [review]

File: EIPS/eip-xxxx.md (hunk 3 above)
Comment (rekmarks) [reply EIPS/eip-xxxx.md:14]: Basically "considering" or "informed by".

Comment (lightclient) [reply EIPS/eip-xxxx.md:14]: Oh okay I see. I recommend using rewording to more clearly explain the relationship between this RPC method and the two referred to here.

**Comment by pedrouid** [comment]
Thanks @lightclient 👍 all requested changes have been included

**Comment by MicahZoltu** [comment]
JSON-RPC specifications are moving out of the EIPs repository.  Tentative plan is to move them into the https://github.com/ethereum/eth1.0-specs/ repository and we are in the process of writing the first one (using OpenRPC for specification format).  Rather than pushing this through the EIP process, it would probably be better to just get it into the new process.  The best place to get involved in this migration is at https://discord.gg/VDESaTNM

**Review by l2ichshop2 — APPROVED** [review]
OhArthit

**Comment by lightclient** [comment]
I'm going to go ahead and close this now since we are no longer doing json-rpc spec work here.

**Pull request closed without merging**